event-based-simulators/main/event_based_simulators.py

```diff
@@ -11,7 +11,6 @@ def current_timestamp(format = "%Y-%m-%dT%H:%M:%S.%f"):
 
 logging.basicConfig(format='%(asctime)s %(name)s:%(message)s', level=logging.INFO)
 log = logging.getLogger("sims")
-log.info(os.environ)
```

Comment:
what is the reason to not log the environ?

Reply:
password.